testem.js

```diff
@@ -15,7 +15,8 @@ let config = {
     "Chrome": [
       "--headless",
       "--disable-gpu",
-      "--remote-debugging-port=9222"
+      "--remote-debugging-port=9222",
+      "--no-sandbox"
```

Comment:
?

Reply:
<img width="1238" alt="screen shot 2018-01-30 at 5 22 52 pm" src="https://user-images.githubusercontent.com/183799/35600381-74ef91d8-05e2-11e8-8d5a-b4afdb10c13a.png">

Reply:
This is required for Travis + Chrome usage (the alternative is to use `sudo: required` in your `.travis.yml` which generally increases CI start time)...

Reply:
I don't understand why such a small change (making `getTemplateIterator` async) broke Travis and necessitates a config change? Is this just a change Travis made suddenly?

Reply:
Looking the build logs the version of chrome is not versioned, it is evergreen. The last successful build was 2 months ago https://travis-ci.org/glimmerjs/glimmer.js/builds/314387954. Also Travis a SAAS and has no semver gurantees. This was recently added to the docs. https://github.com/travis-ci/docs-travis-ci-com/commit/2d65f508acab873efc64665d97342d7da1be34b9